odbc-api/tests/integration.rs

```diff
@@ -2775,12 +2782,12 @@ fn list_tables(profile: &Profile, expected: &str) {
     let conn = profile.setup_empty_table(table_name, &["INTEGER"]).unwrap();
 
     let cursor = conn.tables(None, None, Some(table_name), None).unwrap();
-    let actual = cursor_to_string(cursor);
-
-    assert_eq!(expected, actual);
+    let actual = cursor_to_string(cursor).to_lowercase();
+    assert_eq!(expected.to_lowercase(), actual);
 }
 
 /// List tables for various data sources, using a preallocated statement
+/// Table name comparison is insensitive on Windows
```

Comment:
Same here, if this is required to run on CI, of course it should stay in, but I wonder why it would run on my local windows.